src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -432,12 +432,18 @@
   <data name="ItemsSelected.Text" xml:space="preserve">
     <value>items selected</value>
   </data>
+  <data name="Plural.ItemsSelected.Text" xml:space="preserve">
```

Comment:
The `.Text` suffix is no longer needed and can be removed. We should also be able to remove the other strings that are no longer used.

Reply:
Like this? https://github.com/files-community/Files/pull/15433/commits/589c3dc81ff02ea9a11fd9bb932d9c83c2578b1e

Reply:
Yes

Reply:
We don't exactly have a coding convention for plural strings but I liked how you indicated using a lowercase `p`.

Reply:
> I liked how you indicated using a lowercase `p`

OK maybe try using a lowercase `p` prefix.

> We don't exactly have a coding convention for plural strings

I wrote the method to be universal, although it is now using for plurals